manifests/f/Funkin'Team/FridayNightFunkin'/v2.8/Funkin'Team.FridayNightFunkin'.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Funkin'Team.FridayNightFunkin'
```

```suggestion
PackageIdentifier: FunkinTeam.FridayNightFunkin
```
Best to omit punctuation from Package Identifiers for purposes of automations